tests/tests_suite_test.go

```diff
@@ -32,7 +32,11 @@ import (
 )
 
 func TestTests(t *testing.T) {
-	reporters := []Reporter{reporter.NewKubernetesReporter(os.Getenv("ARTIFACTS"))}
+	artifactsDir := os.Getenv("ARTIFACTS")
+	if artifactsDir == "" {
```

Comment:
The default is already provided by hack/functests.sh
https://github.com/kubevirt/kubevirt/blob/master/hack/functests.sh#L24

By doing this, there won't be a way for not having the reporter logs dumped.